src/entity/active_enum.rs

```diff
@@ -275,6 +275,64 @@ mod tests {
         assert_eq!(format!("{}", DeriveCategory::Small), "'S'");
     }
 
+    #[test]
+    fn display_mode() {
+        // Completely default -> All print values
+        #[derive(PartialEq, Eq, EnumIter, DeriveActiveEnum)]
+        #[sea_orm(rs_type = "i32", db_type = "Integer")]
+        enum DefaultDisplay {
+            One = 1,
+            Two = 2,
+            Three = 3,
+        }
+
+        assert_eq!(format!("{}", DefaultDisplay::One), "1");
+        assert_eq!(format!("{}", DefaultDisplay::Two), "2");
+        assert_eq!(format!("{}", DefaultDisplay::Three), "3");
+
+        // All `label` -> All print label
+        #[derive(PartialEq, Eq, EnumIter, DeriveActiveEnum)]
+        #[sea_orm(rs_type = "i32", db_type = "Integer", display = "label")]
+        enum LabelDisplay {
+            One = 1,
+            Two = 2,
+            Three = 3,
+        }
+
+        assert_eq!(format!("{}", LabelDisplay::One), "'One'");
+        assert_eq!(format!("{}", LabelDisplay::Two), "'Two'");
+        assert_eq!(format!("{}", LabelDisplay::Three), "'Three'");
+
+        // Default with override
+        #[derive(PartialEq, Eq, EnumIter, DeriveActiveEnum)]
+        #[sea_orm(rs_type = "i32", db_type = "Integer")]
+        enum DefaultDisplayOverride {
+            #[sea_orm(display = "label")]
```

Comment:
Actually, I think this override should be the thing the user wants?

So `display = "hello"` should result in 

```rust
assert_eq!(format!("{}", DefaultDisplayOverride::One), "'hello'");
```

Reply:
that relied on the implementation of `Value::to_string`, so...

Reply:
done in 656c777e8c7d33d2eb4f316be3b7627f07d5c72f